example.env

```diff
@@ -11,3 +11,4 @@ SIDEKIQ_WEB_PASSWORD=password
 MAIL_FROM=development@example.com
 CONTACT_EMAIL=development@example.com
 PORT=3000
+S3_BUCKET_NAME=nzsl-share-uat-uploaded-files
```

Comment:
You should probably not include an actual environment detail in an example.env, in case it gets picked up with some actual access keys and used instead of a bucket a person actually wanted to use.